h/accounts/schemas.py

```diff
@@ -153,31 +176,8 @@ def _privacy_accepted_message():
     return privacy_msg
 
 
-class SignupSchema(CSRFSchema):
```

Comment:
`SignupSchema` is rewritten below based on functions that've been extracted from it